man/man8/Makefile.am

```diff
@@ -11,36 +11,43 @@ man8_PODS = \
 	swtpm_cert.pod \
 	swtpm_cuse.pod \
 	swtpm_ioctl.pod \
+	swtpm_localca.pod \
 	swtpm_setup.pod \
 	swtpm_setup.conf.pod \
 	swtpm-create-tpmca.pod \
-	swtpm-localca.pod \
 	swtpm-localca.options.pod \
 	swtpm-localca.conf.pod
 
-man8_MANS = \
+man8_generated_MANS = \
 	swtpm.8 \
 	swtpm_bios.8 \
 	swtpm_cert.8 \
 	swtpm_ioctl.8 \
+	swtpm_localca.8 \
 	swtpm_setup.8 \
 	swtpm_setup.conf.8 \
 	swtpm-create-tpmca.8 \
-	swtpm-localca.8 \
 	swtpm-localca.options.8 \
 	swtpm-localca.conf.8
 
 if WITH_CUSE
-man8_MANS += \
+man8_generated_MANS += \
 	swtpm_cuse.8
 endif
 
+man8_static_MANS = \
+	swtpm-localca.8
+
+man8_MANS = \
+	$(man8_generated_MANS) \
+	$(man8_static_MANS)
+
```

Comment:
This approach mirrors the pattern you previously used in [this change](https://github.com/stefanberger/swtpm/commit/3d75c96537384eafd78943844cbc2179ac70fe65#diff-74234bb6f220e6e709adbeed297b846fa9213d094575941927dd009d37f46b52).